src/vscode-dts/vscode.proposed.scmResourceGroupState.d.ts

```diff
@@ -0,0 +1,31 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+declare module 'vscode' {
+	// https://github.com/microsoft/vscode/issues/192009
```

Comment:
@lszomoru after merged I think you'll need to reopen this issue and add the api-finalized label to make sure we're tracking it.